polygeist/lib/Dialect/Polygeist/Transforms/SYCLRaiseHost.cpp

```diff
@@ -51,19 +51,26 @@ class SYCLRaiseHostConstructsPass
 //===----------------------------------------------------------------------===//
 
 namespace {
-struct RaiseKernelName : public OpRewritePattern<LLVM::GlobalOp> {
+struct RaiseKernelName : public OpRewritePattern<LLVM::AddressOfOp> {
```

Comment:
Should this be called `RaiseGetKernelName` instead, as it generates `SYCLHostGetKernelOp`?
Given that we no longer raise to `SYCLHostKernelNameOp`, should we remove that from SYCL dialect?